hedera-node/hedera-app-spi/build.gradle.kts

```diff
@@ -53,4 +53,7 @@ dependencies {
 
   // Temporarily needed until FakePreHandleContext can be removed
   testFixturesCompileOnly(project(":hedera-node:hedera-token-service"))
+  testFixturesCompileOnly(project(":hedera-node:hedera-networkadmin-service-impl"))
```

Comment:
Does it makes sense to have the dependency as compile only?

Reply:
I deleted this dependency, it's no longer needed